layouts/auto_generated_product.tpl

```diff
@@ -106,7 +108,16 @@
                     <div class="buy-btn-content js-buy-btn-content">
                       {% include "buy-button" %}
                     </div>
-                    {% content bind=product %}
+
+                    <div class="product-cross-page-info">
+                      {%- assign cross_page_info_title = "cross_page_info" | lce -%}
+                      {%- assign cross_page_info_title_tooltip = "content_tooltip_all_pages_same_type" | lce -%}
+                      {% xcontent name="cross-page-info" title=cross_page_info_title title_tooltip=cross_page_info_title_tooltip %}
```

Comment:
This is cross language content area and should be more specific to avoid conflicts.

E.g. `cross-page-info` -> `product-social`